pkg/controller/pravegacluster/pravegacluster_controller_test.go

```diff
@@ -714,5 +714,186 @@ var _ = Describe("PravegaCluster Controller", func() {
 				})
 			})
 		})
+
+		Context("Custom spec with ExternalAccess with annotations and empty domain name", func() {
+			var (
+				client client.Client
+				err    error
+			)
+
+			BeforeEach(func() {
+				annotationsMap := map[string]string{
+					"service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-prefix": "abc",
+				}
+				p.Spec = v1alpha1.ClusterSpec{
+					Version: "0.3.2-rc2",
+					ExternalAccess: &v1alpha1.ExternalAccess{
+						Enabled:    true,
+						Type:       corev1.ServiceTypeClusterIP,
+						DomainName: "",
```

Comment:
I'd prefer you avoid mentioning the domain name here, as most users will do when they don't want  a domain name.

Reply:
done